guides/common/modules/proc_configuring-the-base-operating-system-with-offline-repositories-in-rhel-8.adoc

```diff
@@ -0,0 +1,70 @@
+[id="configuring-the-base-operating-system-with-offline-repositories-in-rhel-8_{context}"]
+= Configuring the Base Operating System with Offline Repositories in RHEL 8
+
+Use this procedure to configure offline repositories for the {RHEL} 8 and {ProjectName} ISO images.
+
+.Procedure
+
+. Create a directory to serve as the mount point for the ISO file corresponding to the base operating system's version.
++
+[options="nowrap"]
+----
+# mkdir /media/rhel8-server
+----
+
+. Mount the ISO image for {RHEL} to the mount point.
++
+[options="nowrap" subs="+quotes"]
+----
+# mount -o loop _rhel8-Server-DVD_.iso /media/rhel8-server
+----
++
+. To copy the ISO file's repository data file and change permissions, enter:
++
+[options="nowrap"]
+----
+# cp /media/rhel8-server/media.repo /etc/yum.repos.d/rhel8-server.repo
+# chmod u+w /etc/yum.repos.d/rhel8-server.repo
+----
+
+. Edit the repository data file and add the `baseurl` directive.
++
+[options="nowrap"]
+----
+[RHEL8-BaseOS]
+name=Red Hat Enterprise Linux BaseOS 8.6.0
```

Comment:
```suggestion
name=Red Hat Enterprise Linux BaseOS
```

the same procedure will be used on 8.7, 8.8 and whatever comes then, so I would not mention the exact RHEL version here.